CMakeLists.txt

```diff
@@ -340,7 +340,10 @@ if(RDK_BUILD_PYTHON_WRAPPERS)
     set(Boost_Python_Lib "python")
   endif()
 
-  target_link_libraries(rdkit_py_base INTERFACE Boost::${Boost_Python_Lib})
+  # we require boost > 1.67 now which makes this a tad easier
```

Comment:
It will be interesting to see if this much simpler version  of finding boost stuff now works.

Reply:
I meant simpler in the sense we don't have to use:

```
  # Try each potential boost-python name until one works
  foreach(Boost_Python_LibN ${Boost_Python_Names})
    find_package(Boost ${RDK_BOOST_VERSION} COMPONENTS "${Boost_Python_LibN}" QUIET)
```

anymore for boost > 1.67